### PR TITLE
[datacenter/ucp] Document RethinkDB Configuration

### PR DESCRIPTION
**Pull request by jlhawn**
### Proposed changes

I decided to document how to get status and manually reconfigure replication of the RethinkDB cluster in Docker Universal Control Plane because there was no mention of it before.

### CODE REVIEW

**Comment by jlhawn** [comment]
@joaofnfernandes please take a look.

File: datacenter/ucp/2.1/guides/admin/monitor-and-troubleshoot/troubleshoot-configurations.md [hunk 1]
```diff
@@ -71,6 +73,44 @@ On failure the command exits with an error code, and no output.
 To learn more about the `etcdctl` utility, check the
 [etcd official documentation](https://coreos.com/etcd/docs/latest/).
 
+## RethinkDB Database
+
+User and organization data for Docker Datacenter is stored in a RetinkDB
+database which is replicated across all conroller nodes in the UCP cluster.
```
Comment (joaofnfernandes) [inline datacenter/ucp/2.1/guides/admin/monitor-and-troubleshoot/troubleshoot-configurations.md]: conroller->controller

File: datacenter/ucp/2.1/guides/admin/monitor-and-troubleshoot/troubleshoot-configurations.md [hunk 2]
```diff
@@ -71,6 +73,44 @@ On failure the command exits with an error code, and no output.
...
+
+Replication and failover of this database is typically handled automatically by
+UCP's own configuration management processes, but detailed database status and
+manual reconfiguration of database replication is available through a command
+line tool available as part of UCP.
+
+The examples below assume you are logged in with ssh into a UCP controller node.
```
Comment (joaofnfernandes) [inline datacenter/ucp/2.1/guides/admin/monitor-and-troubleshoot/troubleshoot-configurations.md]: UCP controller -> UCP manager
I know we use "controller" in some places, but I'm trying to get rid of it now that manager nodes and UCP controllers are 1:1.

**Comment by joaofnfernandes** [comment]
Aside from the nitpicks, this looks awesome. Thanks for investing the time to do this 🚀

File: datacenter/ucp/2.1/guides/admin/monitor-and-troubleshoot/troubleshoot-configurations.md (hunk 2 above)
Comment (jlhawn) [inline datacenter/ucp/2.1/guides/admin/monitor-and-troubleshoot/troubleshoot-configurations.md]: I've gone through and updated the document to remove the use of the word "controller" (both the ones I added and those that already existed).

**Comment by jlhawn** [comment]
@joaofnfernandes Is the Jenkins failure something I should be concerned with?

**Comment by joaofnfernandes** [comment]
I saw some changes to our build system go by in the last couple of days, so try rebasing.

**Pull request merged**